client/src/pages/Home.css

```diff
@@ -9,3 +9,46 @@
 .message:hover {
 	background-color: yellow;
 }
+
+.flex-container {
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(214, 227, 243);
+  align-items: center;
+  height: 100%;
+  border-radius: 12px;
+  padding: 100px 100px 100px 100px;
+  width: 70%;
```

Comment:
We shouldn't give this element a specific width because then it's too large for a mobile view — let's delete it

<img src=https://user-images.githubusercontent.com/6434329/150690610-aa268246-9b9a-4cf4-bc95-a662e05a029e.png width=300>